cli/Valet/Site.php

```diff
@@ -631,8 +631,11 @@ public function buildCertificateConf(string $path, string $url): void
     public function buildSecureNginxServer(string $url, string $siteConf = null): string
     {
         if ($siteConf === null) {
+            $nginxVersion = ltrim(exec('nginx -v'), 'nginx version: nginx/');
```

Comment:
Since `ltrim()` technically doesn't support passing a "phrase" (it treats it as a bunch of characters, not a phrase), perhaps simply `str_replace()` might be better?

Reply:
Looks good. Thx. 👍

Reply:
@OpJePl44tsm4n perhaps also rework the Title of this PR slightly to incorporate "http2" into it?
Maybe something like "handle deprecated http2_push_preload conf for nginx >= 1.25.1" ?